package.json

```diff
@@ -208,10 +208,11 @@
     "@hapi/subtext": "^7.0.3",
     "@kubernetes/client-node": "^0.16.3",
     "@material-ui/styles": "^4.11.5",
-    "@ogre-tools/fp": "7.1.0",
-    "@ogre-tools/injectable": "7.1.0",
-    "@ogre-tools/injectable-extension-for-auto-registration": "7.1.0",
-    "@ogre-tools/injectable-react": "7.1.0",
+    "@ogre-tools/fp": "8.0.0",
+    "@ogre-tools/injectable": "8.0.0",
+    "@ogre-tools/injectable-extension-for-auto-registration": "8.0.0",
+    "@ogre-tools/injectable-react": "8.0.0",
+    "@ogre-tools/injectable-extension-for-mobx": "8.0.0",
```

Comment:
Can you add these via `yarn add` and not manually because these are not in sorted order which messes up other additions.

Reply:
Done